include/devicetree.h

```diff
@@ -2471,6 +2471,26 @@
  */
 #define DT_INST_LABEL(inst) DT_INST_PROP(inst, label)
 
+/**
+ * @brief Get a string property's value as a token.
+ * @param inst instance number
+ * @param prop lowercase-and-underscores property string name
+ * @return the value of @p prop as a token, i.e. without any quotes
+ *         and with special characters converted to underscores
+ */
+#define DT_INST_STRING_TOKEN(inst, prop) \
+	DT_STRING_TOKEN(DT_DRV_INST(inst), prop)
+
+/**
+ * @brief Like DT_INST_STRING_TOKEN(), but uppercased.
+ * @param inst instance number
+ * @param prop lowercase-and-underscores property string name
+ * @return the value of @p prop as a token, i.e. without any quotes
+ *         and with special characters converted to underscores
```

Comment:
Can you please apply the same fix I sent here? https://github.com/zephyrproject-rtos/zephyr/pull/44237